pkg/plugin/internal/util/go_version.go

```diff
@@ -69,7 +69,7 @@ func checkGoVersion(verStr string) error {
 	}
```

Comment:
Go version checks should be refactored into versioned plugin packages such that the binary can accept multiple go versions.

Reply:
good catcher.